pkg/schedule/checker/replica_checker.go

```diff
@@ -61,6 +61,10 @@ func (*ReplicaChecker) Name() string {
 	return types.ReplicaChecker.String()
 }
 
+func (*ReplicaChecker) GetType() types.CheckerSchedulerType {
```

Comment:
```suggestion
// GetType ...
func (*ReplicaChecker) GetType() types.CheckerSchedulerType {
```

Reply:
Filed an issue https://github.com/tikv/pd/issues/8458#issue-2437331188

And fixed it https://github.com/tikv/pd/pull/8440/commits/0d2676fc4c3f82a3a9a8123e9c2d2529aaa2e3b5